src/overlays/actors/ovl_Bg_F40_Switch/z_bg_f40_switch.c

```diff
@@ -34,37 +34,167 @@ const ActorInit Bg_F40_Switch_InitVars = {
     (ActorFunc)BgF40Switch_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80BC4E04[] = {
+s32 sBgF40SwitchGlobalsInitialized = false;
+u32 sBgF40SwitchLastUpdateFrame;
+
+extern CollisionHeader object_f40_switch_Colheader_000118;
+extern Gfx object_f40_switch_DL_000438;
+extern Gfx object_f40_switch_DL_000390;
```

Comment:
You should remove those externs from here and include the object instead. You should only need to do `#include "objects/object_f40_switch/object_f40_switch.h"` at the top of the file.

If you feel adventurous enough, you could open that object in Z64Utils, see what those symbols are and try to name them, but it isn't required. Up to you

Reply:
Also, don't forget to remove the symbols from undefined_syms.txt